src/style.css

```diff
@@ -1 +1,98 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
+  src: url('./fonts/Avenir-Heavy.ttf');
+  font-weight: 400;
+}
+
+html {
+  font-weight: 300;
+}
+
+body {
+  margin: 0;
+}
+
+.searchBar {
+  display: flex;
+  position: absolute;
+  align-items: center;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  width: 100%;
+  margin-left: 48px;
+}
+
+.searchBar--big {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  margin-left: 17px;
+}
+
+.searchBar--small {
+  margin-left: 17px;
+  top: 90px;
+  height: 42px;
+}
+
+.searchBar__input {
+  width: 100%;
+  display: block;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
Good job specifying fallback fonts here. However, it would be even better to include a generic family name at the end of the list as the final fallback, for example, `sans-serif`. This ensures that if none of the specified fonts are available, the browser will display a default font from the generic family.